soccer/src/soccer/ui/main_window.cpp

```diff
@@ -1170,3 +1167,30 @@ void MainWindow::updateDebugLayers(const LogFrame& frame) {
         _ui.debugLayers->sortItems();
     }
 }
+
+void MainWindow::on_addToTable_clicked() {
+    auto to_add = (_ui.testInput->toPlainText().toStdString());
+    boost::trim(to_add);
+    auto* selectedTestsTable = _ui.selectedTestsTable;
```

Comment:
nit: should be snake case 

`selected_tests_table`

Reply:
in the .ui files they use camelCase for some reason, so I was being consistent. I just got rid of this temp var though.